components/cluster/resource-agents-openindiana/files/IPaddr

```diff
@@ -0,0 +1,934 @@
+#!/usr/gnu/bin/sh
```

Comment:
I didn't check it out if it is POSIX compatible.
Script is copied from the linux world and there is sh=bash.